src/i18n/langs/en.json

```diff
@@ -277,15 +277,15 @@
       "no_and_shutdown": "No and Shutdown",
       "ok": "OK",
       "other_options": "Other Options",
-      "password_a_details": "Use this password for login to your RaspiBlitz.",
+      "password_a_details": "Also known as Password A. Use this password for login to your RaspiBlitz.",
```

Comment:
I played around with that so we can add the coloring like the mockups.

See https://react.i18next.com/latest/trans-component#samples

Reply:
Yeah, about that...
I think it's a bit too close to a link tbh.

Link:
<img width="478" alt="image" src="https://github.com/raspiblitz/raspiblitz-web/assets/1016218/99d39bd6-f54c-4c41-96ca-524bda789833">

Strong:
<img width="416" alt="image" src="https://github.com/raspiblitz/raspiblitz-web/assets/1016218/def7bf0a-769e-4960-9bab-72f6a73dbfeb">

wdyt? Is this an issue?

Reply:
Would rather change the link color as you did yes 

The other passwords have other colors so it will differ from a link

Reply:
Wait, I thought those were color variants but we decided to go with blue :D So each color is different? Ahhhh, ok.

Should we underline "real" links? I like that. Makes it obvious.

Reply:
Yes, the color variants are different so it makes a difference for the user. :)

Yes, links should be unterlined IMO: https://nextui.org/docs/components/link#underline

Maybe we can put that into the config or something to it always applies?

Reply:
done